opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -103,23 +103,31 @@ impl UserEventsExporter {
         event_sets
     }
 
-    fn add_attribute_to_event(&self, eb: &mut EventBuilder, (key, value): (&Key, &AnyValue)) {
+    fn add_attribute_to_event(
+        &self,
+        eb: &mut EventBuilder,
+        (key, value): (&Key, &AnyValue),
+    ) -> bool {
         let field_name = key.as_str();
         match value {
             AnyValue::Boolean(b) => {
                 eb.add_value(field_name, *b, FieldFormat::Boolean, 0);
+                true
             }
             AnyValue::Int(i) => {
                 eb.add_value(field_name, *i, FieldFormat::SignedInt, 0);
+                true
             }
             AnyValue::Double(f) => {
                 eb.add_value(field_name, *f, FieldFormat::Float, 0);
+                true
             }
             AnyValue::String(s) => {
                 eb.add_str(field_name, s.as_str(), FieldFormat::Default, 0);
+                true
             }
             // TODO: Handle other types. Arrays are required in Trace for storing Links.
-            _ => (),
+            _ => false,
```

Comment:
How about we add an empty string as value? That way we would at least send information about the key.

Reply:
yea that'll make less silent drop!. Added